lib/trento/users.ex

```diff
@@ -76,18 +78,18 @@ defmodule Trento.Users do
     |> Repo.insert()
   end
 
-  def update_user_profile(%User{id: 1}, _), do: {:error, :forbidden}
-
-  def update_user_profile(%User{} = user, attrs) do
-    updated_attrs = maybe_set_password_change_requested_at(attrs, true)
+  def update_user_profile(%User{username: username} = user, attrs) do
```

Comment:
This change is so painful. It was so nice with that readable pattern matching :sweat_smile: 
Could we maybe try to experiment with some macro magic?
Or assign the value to a constant? 
Or something like that.
I don't know.
This change makes me sad :stuck_out_tongue_closed_eyes:

Reply:
It's ugly has hell, but the value is set at runtime, so we can't use compile_env, or some costant in the pattern matching, even with macros the value is not known at compile time